Bottom: 1ece294614eede2ff73f68d0f1850bcbc6f467a8
Top:    7169246c1dcdabd9c359682283ebe77b7f4baf40
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 18:53:17 +0000

Refresh of autoroute-c-if-0-out-an-unused

---

diff --git a/src/autoroute.c b/src/autoroute.c
index ea97d01..9ccab97 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -3428,6 +3428,7 @@ RD_DrawManhattanLine (routedata_t * rd,
 }
 
 /* for smoothing, don't pack traces to min clearance gratuitously */
+#if 0
 static void
 add_clearance (CheapPointType * nextpoint, const BoxType * b)
 {
@@ -3464,6 +3465,7 @@ add_clearance (CheapPointType * nextpoint, const BoxType * b)
 	nextpoint->Y = (b->Y1 + b->Y2) / 2;
     }
 }
+#endif
 
 /* This back-traces the expansion boxes along the best path
  * it draws the lines that will make the actual path.
